chapters/tutorials/index.rst

```diff
@@ -34,45 +34,48 @@ And we need the following dependencies:
-Once that is ready, we can encode our data into vectors and finally store those vectors, so it is ready for indexing and querying.
+Once we have our data, as usual in machine learning, you'll probably need to pre-process that data. To keep this as simple as possible for this first tutorial, we won't need to do any pre-processing. But remember that this is a possibility for another use case.
+Once that is ready, we can encode our data into vectors and finally store those vectors, so they are ready for indexing and querying.
 
 Tutorial
 ---------
 
-Define data and work directory
+Define data and work directories
 ++++++++++++++++++++++++++++++++++++
 
 We can start creating an empty folder, I'll call mine `tutorial` and that's the name you'll see through the tutorial but feel free to use whatever you wish.
 
-We will display our results in our browser, so download the `static` folder from `here <https://github.com/jina-ai/jina/tree/master/jina/helloworld/chatbot/static>`_, and paste it in in your `tutorial` folder. This is only the CSS and HTML files to render our results. We will use a dataset in a .csv format. I'll use the `COVID <https://www.kaggle.com/xhlulu/covidqa>`_ dataset from Kaggle. You don't need to download this by hand, we'll do it later in our app.
+We will display our results in our browser, so download the `static` folder from `here <https://github.com/jina-ai/jina/tree/master/jina/helloworld/chatbot/static>`_, and paste it into your `tutorial` folder. This is only the CSS and HTML files to render our results. We will use a dataset in a .csv format. I'll use the `COVID <https://www.kaggle.com/xhlulu/covidqa>`_ dataset from Kaggle. You don't need to download this by hand, we'll do it later in our app.
 
 Create a Flow
 ++++++++++++++++++++++++++++++++++++
 
-To create a Flow you only need to import it from Jina:
+The very first concept you'll see in Jina is a `Flow`. You can see `here <https://github.com/jina-ai/jina/blob/master/.github/2.0/cookbooks/Flow.md>`_ a more formal introduction of what it is, but for now, think of the `Flow` as a manager in Jina, it takes care of the all the tasks that will run on your application and each `Flow` object will take care of one real-world task.
+
+To create a `Flow` you only need to import it from Jina.
+So open your favorite IDE and let's start writing our code:
 
 .. code-block:: python
 
     from jina import Flow
-    f = Flow()
+    flow = Flow()
 
-But this is an empty Flow, since we want to encode our data and then index it, we need to add elements to it.
+But this is an empty `Flow`, since we want to encode our data and then index it, we need to add elements to it. And what we add to a `Flow` are always `Executors` we will talk about them more formally later. But think of them as the elements you will need to do all the processing you want.
```

Comment:
But this is just an empty `Flow`. Since we want to encode our data and then index it, we need to add elements to it.

(note punctuation)